app/helpers/user_helper.rb

```diff
@@ -23,8 +23,8 @@ def moderation_view?
 
   private
 
-  def profile_link(user)
-    link_to(user.profile.safe_name, user_path(user), class: ("user--banned" if user.banned?).to_s)
+  def profile_link(user, target = nil)
```

Comment:
would be cleaner to have this as a keword argument
```suggestion
  def profile_link(user, target: nil)
```